### DISCUSSION TITLE
Scan All button doesen't work

### DISCUSSION THREAD

**Discussion by lazycatze** [photoview/photoview#763] [Q&A and Troubleshooting]
Hi folks,

I really hate to bother you all with lame questions, but I'm a brand-new Docker (Portainer) user, so I hope someone will help me.

I installed Photoview to my Asustor NAS via Portainer. The path to the photos is correct, it looks this way:
/volume1/home/admin/Photo & Video

The trouble is that I cannot scan it - there are no errors just this message: 'Scanner complete
All jobs have been scanned' - but no data is displayed.

I read the manual on site with advice to check some Docker settings. But I have no clue what I have to do to fix it. As I mentioned above I'm using Portainer at my Asustor.

I hope for your help, folks. Thanks.

**Comment by stephenp** [comment]
I'm not familiar with Portainer, but maybe there's an issue parsing the `&` and spaces in your file path?  Are there any other messages in the log when you do a scan that could shed more light onto possible issues?

**Reply by lazycatze** [reply]
Thanks for your answer.

Well I double-checked all my moving and it was my mistake - I used the full way to the data directory instead /photos. It works well now.
